src/assets/scripts/client/constants/themes/default/modules/terrain.js

```diff
@@ -0,0 +1,32 @@
+export const TERRAIN_THEME = {
+    /**
+     * Opacity of the lines on outer edges of terrain-elevated areas
+     *
+     * @memberof TERRAIN_THEME
+     * @property BORDER_OPACITY
+     */
+    BORDER_OPACITY: 1,
+
+    /**
+     * Opaxity of the fill color within terrain-elevated areas
+     *
+     * @memberof TERRAIN_THEME
+     * @property FILL_OPACITY
+     */
+    FILL_OPACITY: 0.1,
+
+    /**
+     * Colors for each elevation level (in feet), in RGB
+     *
+     * @memberof TERRAIN_THEME
+     * @property COLOR
+     */
+    COLOR: {
```

Comment:
I feel the inline object here should be split out to its own definition somewhere else.

could you also add a `TODO` here to refactor the consumer(s) of this value to use actual `rgb` values? or at the very least accept `rgb` values and build the data it needs from those.  That way we can stick with the colors defined for a theme and the consumer can use those values.